sanic/server/loop.py

```diff
@@ -1,12 +1,26 @@
 import asyncio
 
-from distutils.util import strtobool
 from os import getenv
 
 from sanic.compat import OS_IS_WINDOWS
 from sanic.log import error_logger
 
 
+def strtobool(query: str) -> bool:
```

Comment:
Why do we need to do it twice inside the lib?

Reply:
we don't, I missed updating this to use sanic.utils